### PR TITLE
feat: :sparkles: add alt labels to navbar logos

### PR DESCRIPTION
**Pull request by signekb**

### CODE REVIEW

**Review by K-Beicher — DISMISSED** [review]
Even better :-)

File: _quarto.yml [hunk 1]
```diff
@@ -16,13 +16,16 @@ website:
   navbar:
     title: false
     logo: _extensions/seedcase-project/seedcase-theme/logos/navbar-logo-seedcase-decisions.svg
+    logo-alt: "Seedcase Decisions logo: Go to main page"
```
Comment (signekb) [inline _quarto.yml:19]:
```suggestion
    logo-alt: "Seedcase Decisions logo: Main page"
```

File: _quarto.yml [hunk 2]
```diff
@@ -16,13 +16,16 @@ website:
...
     pinned: true
     search: true
     tools:
       - icon: house
         href: "https://seedcase-project.org"
+        aria-label: "House icon: Go to the home page of the Seedcase Project"
```
Comment (signekb) [inline _quarto.yml:25]:
```suggestion
        aria-label: "House icon: Seedcase Project home page "
```

File: _quarto.yml [hunk 3]
```diff
@@ -16,13 +16,16 @@ website:
...
       - icon: github
         href: "https://github.com/seedcase-project/decisions"
+        aria-label: "GitHub icon: Go to the source code on GitHub"
```
Comment (signekb) [inline _quarto.yml:28]:
```suggestion
        aria-label: "GitHub icon: Source code"
```

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**